finance/templates/finance/get_finance.html

```diff
@@ -47,10 +47,10 @@
 
 <section id="advantages-section">
     <div class="container">
+        <div class="column-one-half">
```

Comment:
why did this move out of the grid-row?

Reply:
oh i see now. you should put this in its own grid-row then you won't need the clearfix div